### ISSUE TITLE
merging two field into one

### ISSUE THREAD

**Issue by ordinaryman09** [matthewcheok/Realm-JSON#91]
hello,

is there any way to merge two field into one?
for example, if the dictionary contains the following:
{ street 1 : "xxx",
street 2 : "yyy" }

I want merge it to just "street" property. 

thanks!

**Comment by viktorasl** [comment]
Hey. You can use static `jsonPreprocessing:` method and merge those fields before persisting the object

**Comment by ordinaryman09** [comment]
Thank you.
I check your code a little bit more and I noticed the selector you're looking for is **preprocessedJSON** not jsonPreprocessing. Also, I think it might be helpful for others if you update the ReadMe

```
+ (NSDictionary *)preprocessedJSON:(NSDictionary *)dictionary {
    NSMutableDictionary *dict = [[NSMutableDictionary alloc] initWithDictionary:dictionary];
    dict[@"releaseCount"] = @(0);
    return dict.copy;
}
```

**Comment by viktorasl** [comment]
Yes, sorry, the method is called `preprocessedJSON:`. PR is already made #93

**Comment by ordinaryman09** [comment]
No worries. great work! Thanks brother.
